src/Blocks/components/form/assets/conditional-tags.js

```diff
@@ -0,0 +1,562 @@
+
+import { State } from './state';
+import {
+	CONDITIONAL_TAGS_OPERATORS,
+	CONDITIONAL_TAGS_ACTIONS,
+	CONDITIONAL_TAGS_LOGIC,
+} from '../../conditional-tags/assets/utils';
+import { prefix, setStateWindow } from './state/init';
+
+/**
+ * Main conditon tags class.
+ */
+export class ConditionalTags {
+	constructor() {
+		this.state = new State();
+
+		// Simplify usage of constants
+		this.SHOW = CONDITIONAL_TAGS_ACTIONS.SHOW;
+		this.HIDE = CONDITIONAL_TAGS_ACTIONS.HIDE;
+		this.OR = CONDITIONAL_TAGS_LOGIC.OR;
+		this.AND = CONDITIONAL_TAGS_LOGIC.AND;
+
+		// Map all conditional logic as a object.
+		this.OPERATORS = {
+			[CONDITIONAL_TAGS_OPERATORS.IS]: (input, value) => value === input,
+			[CONDITIONAL_TAGS_OPERATORS.ISN]: (input, value) => value !== input,
+			[CONDITIONAL_TAGS_OPERATORS.GT]: (input, value) => parseFloat(String(input)) > parseFloat(String(value)),
+			[CONDITIONAL_TAGS_OPERATORS.GTE]: (input, value) => parseFloat(String(input)) >= parseFloat(String(value)),
+			[CONDITIONAL_TAGS_OPERATORS.LT]: (input, value) => parseFloat(String(input)) < parseFloat(String(value)),
+			[CONDITIONAL_TAGS_OPERATORS.LTE]: (input, value) => parseFloat(String(input)) <= parseFloat(String(value)),
+			[CONDITIONAL_TAGS_OPERATORS.C]: (input, value) => input.includes(value),
+			[CONDITIONAL_TAGS_OPERATORS.SW]: (input, value) => input.startsWith(value),
+			[CONDITIONAL_TAGS_OPERATORS.EW]: (input, value) => input.endsWith(value),
+		};
+
+		// Set all public methods.
+		this.publicMethods();
+	}
+
+	////////////////////////////////////////////////////////////////
+	// Public methods
+	////////////////////////////////////////////////////////////////
+	/**
+	 * Init one form by element after the loaded event is fired.
+	 * 
+	 * @param {string} fromId Form Id.
+	 *
+	 * @returns {void}
+	 */
+	initOne(formId) {
+		if (this.state.getStateConfigIsAdmin(formId)) {
+			return;
+		}
+
+		this.state.getStateFormElement(formId).addEventListener(
+			this.state.getStateEventsFormJsLoaded(),
+			this.onInitEvent
+		);
+	}
+
+	/**
+	 * Init forms conditional logic.
+	 *
+	 * @param {string} fromId Form Id.
+	 *
+	 * @returns {void}
+	 */
+	initForms(formId) {
+		// Get all tags from state.
+		let tags = this.state.getStateFormConditionalTagsForm(formId);
+		if (!tags) {
+			return;
+		}
+
+		// Loop tags.
+		tags.forEach((tag) => {
```

Comment:
Maybe to extract tag to be more readable;


```suggestion
		tags.forEach(([tagId, tagState, tagSomething]) => {
```

Reply:
awesome tnx